sandbox/sandbox/SceneDelegate.swift

```diff
@@ -15,7 +15,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             // the Unity window behind it.
             vc.view.isOpaque = false
             vc.view.backgroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.0)
-            vc.view.tag = UIWindowCustom.PassthroughTag
+            
+            // This appears to not allow the ColorPicker to function.
+            //vc.view.tag = UIWindowCustom.PassthroughTag
```

Comment:
Any reason why this is not needed anymore?

I don't really use Unity anymore. I guess I could try to run a build with this MR just to be sure I merge something that works for everyone